public/Luadoc/LuaDocumentation.xml

```diff
@@ -4762,11 +4836,11 @@ prev_note_name, succeeded = options:NoteSkin("cel")
 		Returns the profile for player <code>pn</code>.
 	</Function>
 	<Function name='GetProfileDir' return='string' arguments='ProfileSlot slot'>
-		Returns the profile directory for the provided <Link class="ENUM" function="ProfileSlot">ProfileSlot</Link>, formatted like <code>/Save/LocalProfiles/00000001/</code>.<br />
-		Returns an empty string if the provided <Link class="ENUM" function="ProfileSlot">ProfileSlot</Link> does not have a <Link class="Profile" /> loaded.
+		Returns the profile directory for the provided <Link class='ENUM' function='ProfileSlot'>ProfileSlot</Link>, formatted like <code>/Save/LocalProfiles/00000001/</code>.<br />
+		Returns an empty string if the the provided <Link class='ENUM' function='ProfileSlot'>ProfileSlot</Link> does not have a <Link class='Profile' /> loaded.

```

Comment:
extra `the`

also `does not have a loaded.` (I think should be `is not loaded` ?)

Reply:
Good catch on the extra `the`.  I'll remove that.

---

> also `does not have a loaded.` (I think should be `is not loaded` ?)

That text gets rendered as

> Returns an empty string if the the provided [ProfileSlot](https://itgmania.github.io/lua-For-itgmania/LuaAPI#Enums-ProfileSlot) does not have a [Profile](https://itgmania.github.io/lua-For-itgmania/LuaAPI#Classes-Profile) loaded.

<img width="602" alt="Screen Shot 2022-06-20 at 10 23 11 AM" src="https://user-images.githubusercontent.com/1253483/174623322-5345e32d-35d0-4d96-acd5-c13be477b03e.png">

so it's as-intended.

Reply:
OH I misread. Thanks for the clarification!